### ISSUE TITLE
[BUG]  Keys are getting stuck

### ISSUE THREAD

**Issue by PedrixzZ** [PojavLauncherTeam/PojavLauncher#4469]
### Describe the bug

.

### The log file and images/videos

.

### Steps To Reproduce

```markdown
1. Start PojavLauncher
2. Start Minecraft
3. Enter World
```

### Expected Behavior

The controls are all buggy. As in the video: https://www.mediafire.com/file/gr7jgwthhm5y7ax/screen-20230808-234357~2.mp4/file

### Platform

```markdown
- Device model: Moto e20
- CPU architecture: arm
- Android version: 11
- PojavLauncher version: dahlia-20230813-bdb25c1-v3_openjdk
```

### Anything else?

even if I let go of the screen the buttons stay clicked

**Comment by GaucovyAero** [comment]
known issue

**Comment by PedrixzZ** [comment]
Will it be fixed?

**Comment by PedrixzZ** [comment]
@Mathias-Boulay @GaucovyAero

**Comment by rsxramin** [comment]
Have the same issue on Galaxy Tab S7FE(Snapdragon). Works fine on Galaxy A54 and S23FE tho(both Exynos). It's literally unplayable so idk why it hasn't been fixed yet. Fps is fine, upwards of 120 with sodium on 1.21(Modded). Occurs in both singleplayer and multiplayer.

**Comment by Adzetko** [comment]
I can confirm I have this bug too, on a Redmi Note 10 Pro (codename: sweet). The only workaround I know of is to restart the instance. While waiting for this to get fixed, does anyone have a more convenient workaround?

**Comment by CowSheep72** [comment]
me too (Xiaomi 11T Pro)

**Comment by CowSheep72** [comment]
however it was only after foxglove release

**Comment by Mathias-Boulay** [comment]
If anyone of you can consistently reproduce it, then I'll be very happy to address it. Sadly none of us can't, making it impossible to tackle the issue

**Comment by Mathias-Boulay** [comment]
@CowSheep72 @Adzetko @rsxramin @PedrixzZ

**Comment by CowSheep72** [comment]
imma try to reproduce later @Mathias-Boulay

**Comment by Fede2782** [comment]
Can confirm on A346B, Androis 14, mediatek dimensity 1080, happens too

**Comment by jelyfzh** [comment]
i have the same bug on the galaxy tab A9+ 128GB 

EDIT: the bug does not appear on my device in version 1.7.10, but does in 1.20.6. have not checked other versions yet

**Comment by Mathias-Boulay** [comment]
Finally addressed by 366f1815331bd9d7e592ac9e021b387bd2ed99fa
